utilities.lib.nut

```diff
@@ -157,28 +183,20 @@ utilities._isLeapYear <- function(y) {
 // **********         Public           **********
 utilities.getNewUUID <- function(cb = null) {
     if (cb == null) {
-        server.error("getNewUUID() requires a callback function with err, data parameters");
-        return;
+        throw "getNewUUID() requires a callback function with err, data parameters";
     }
 
     ::_uuidcb <- cb;
```

Comment:
:: _uuidcb
creating a global variable is not ideal for a library.
If it needs to be a global it should at the very least be name-spaced with utilities and/or noted in the docs. 
Better yet make this a slot in the utilities table and use it like a class variable would be used.